tests/ibm_appscan_test.py

```diff
@@ -0,0 +1,93 @@
+from selenium import webdriver
```

Comment:
A selenium is good, but a little slow compared to what could be done. Check out this [parser test](https://github.com/propersam/django-DefectDojo/blob/9280eb868d8eaadba4f6fd02676680393f73cf96/dojo/unittests/test_immuniweb_parser.py).

Reply:
I wanted to use a parser test... but the sample file of IBM APP Scan is very large

Reply:
Can it be reduced in size?

Reply:
I don't know how possible that is...

IBMApp Scanner xml result is a really sophisticated xml file by design.

Reply:
I've checked it out and you're right. There's not much that can be done about this. Great work making a parser from that mess of a file.

Reply:
yea... thanks